### ISSUE TITLE
Can't make this work

### ISSUE THREAD

**Issue by Damianoatlante** [wickercar/foundry-ai-text-importer#2]
![Screenshot 2024-02-11 160845](https://github.com/wickercar/foundry-ai-text-importer/assets/119633900/11f3ed03-9571-47a6-98c2-225b46b39780)

Hi, I'm trying to make the module work but I only get this error. I generated an API on OpenAI and pasted it but It doesn't seem to work

**Comment by wickercar** [comment]
Hey @Damianoatlante , thanks for calling this out! To use the current implementation, you must have access to GPT-4 (including gpt-4-turbo-preview) or else the module will not work. I tried accomplishing this import in several different ways using gpt-3.5-turbo, but ultimately the context window is too small to effectively parse the monster block.

This was definitely not clear though! In https://github.com/wickercar/foundry-ai-text-importer/commit/8a7987cba100dd89a57b4a5bb3258cac12e6814e I added a new invalidation state to let a user know that their key is valid but does not have access to the correct models. 

Will close once this is released! Thanks for trying it out
